smartsim/_core/control/controller.py

```diff
@@ -346,6 +347,28 @@ def init_launcher(self, launcher: str) -> None:
         else:
             raise TypeError("Must provide a 'launcher' argument")
 
+    @staticmethod
+    def symlink(
```

Comment:
Should this function be called `symlink_output` or something similar? Otherwise it may seem like we are symlinking the `Step` and the `Entity`.

Reply:
Great point. Fixing!